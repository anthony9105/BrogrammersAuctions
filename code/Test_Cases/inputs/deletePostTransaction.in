login adminUser1 ^^^^ 
delete 
nonAdminUser1
logout
close