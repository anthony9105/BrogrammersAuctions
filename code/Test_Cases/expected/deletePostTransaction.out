Welcome to the Auction System. 
Please login to begin. 
Welcome, adminUser1.
Delete nonAdminUser1
Session complete.
Username is unknown